src/views/transaction/components/transaction-form/transaction-form.view.jsx

```diff
@@ -367,6 +368,7 @@ function TransactionForm ({
         const transactionFee = getFee(fees, true)
         return onSubmit({
           amount: amount,
+          from: { accountIndex: account.accountIndex },
```

Comment:
Isn't this only necessary for transfers and other txs should go to the home page?

Reply:
Yep, I also think that we don't need the `accountIndex here`